pkg/serviced.default

```diff
@@ -1,194 +1,247 @@
+# for the “metadata” device which is used for the thin pool. The default size is 2G.  The file is sparse,
+# so it will not initially take up this much space.
+# SERVICED_DM_LOOPMETADATASIZE=2G
 
-# Disable TLS for RPC connection, TLS is enabled by default
-# SERVICED_RPC_DISABLE_TLS=false
+# Device mapper dm.thinpooldev parameter.  Specifies a custom block storage device to use for the thin pool.
+# SERVICED_DM_THINPOOLDEV=
 
-# Set the minimum supported TLS version for RPC connections, valid values VersionTLS10|VersionTLS11|VersionTLS12
-# SERVICED_RPC_TLS_MIN_VERSION=VersionTLS10
+# Docker DNS configuration used for running containers.  Multiple values should be comma separated.
+# SERVICED_DOCKER_DNS=
 
-# Set the supported TLS ciphers for RPC connections
-# SERVICED_RPC_TLS_CIPHERS=TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+# Comma-separated list of key=value options, corresponding to Docker --log-opt options.
+# For log drivers which do not need additional arguments (e.g. journald), use a single comma as the value
+# of SERVICED_DOCKER_LOG_CONFIG and the system will use the defaults for that type of log driver.
+# SERVICED_DOCKER_LOG_CONFIG=max-file=5,max-size=10m
 
-# Set the UI port address to use: port, :port, ip:port
-# SERVICED_UI_PORT=:443
+# Specify the log driver for all docker containers logs, including isvc containers on the master node.
+# Direct port of Docker --log-driver option. Values include json-file, syslog, journald, gelf, fluentd, and none.
+# SERVICED_DOCKER_LOG_DRIVER=json-file
 
-# Set the frequency in seconds that the UI will poll serviced for updates
-# SERVICED_UI_POLL_FREQUENCY=3
+# Set the local docker registry
+# SERVICED_DOCKER_REGISTRY=localhost:5000
 
-# Set the mux port to listen on
-# SERVICED_MUX_PORT=22250
+# Set the default serviced RPC endpoint to dial
+# SERVICED_ENDPOINT={{SERVICED_MASTER_IP}}:4979
 
-# Disable TLS for muxed connections. TLS is enabled by default
-# SERVICED_MUX_DISABLE_TLS=0
+# Time (in seconds) to wait for elastic search to start
+# SERVICED_ES_STARTUP_TIMEOUT=240
 
-# Set the minimum supported TLS version for MUX connections, valid values VersionTLS10|VersionTLS11|VersionTLS12
-# SERVICED_MUX_TLS_MIN_VERSION=VersionTLS10
+# The default path for configuration files.  If not set, this will be the path given for SERVICED_HOME
+# followed by /etc
+# SERVICED_ETC_PATH=/opt/serviced/etc
 
-# Set the supported TLS ciphers for MUX connections
-# SERVICED_MUX_TLS_CIPHERS=TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+# Set the driver type on the master for the distributed file system (rsync/btrfs/devicemapper)
+# SERVICED_FS_TYPE=devicemapper
+
+# The default home directory.
```

Comment:
This doesn't seem quite descriptive enough. It's the directory relative to which serviced will put all its stuff.